PayPalCheckout/src/main/java/com/paypal/android/checkout/BrowserSwitchHelper.kt

```diff
@@ -0,0 +1,40 @@
+package com.paypal.android.checkout
+
+import android.net.Uri
+import androidx.annotation.NonNull
+import com.paypal.android.core.CoreConfig
+import com.paypal.android.core.Environment
+import com.braintreepayments.api.BrowserSwitchOptions
+import org.json.JSONObject
+
+
+internal class BrowserSwitchHelper(private val urlScheme: String) {
+
+    private val redirectUriPayPalCheckout = String.format("%s://x-callback-url/paypal-sdk/paypal-checkout", urlScheme)
+
+    private fun buildPayPalCheckoutUri(@NonNull orderId: String?, @NonNull config: CoreConfig): Uri {
+        val baseURL = when (config.environment) {
+            Environment.LIVE -> "https://www.paypal.com"
```

Comment:
In iOS we pulled these URLs into an Environment extension in the PayPal module - this doesn't exist in Kotlin, do we think these are ok to live in this helper class or should they go elsewhere?